src/main/java/lt/vu/web/api/domain/order/CreateOrder.java

```diff
@@ -0,0 +1,78 @@
+package lt.vu.web.api.domain.order;
+
+import lombok.Getter;
+import lombok.Setter;
+import lt.vu.persistence.orm.entities.*;
+import lt.vu.persistence.orm.repository.OrderRepository;
+import lt.vu.persistence.orm.repository.PackageOptionRepository;
+import lt.vu.web.api.dto.postDtos.order.OrderDTO;
+import lt.vu.web.api.dto.postDtos.order.SenderDTO;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@RequestScoped
+@Getter @Setter
+public class CreateOrder {
+
+    @Inject
+    private OrderRepository orderRepository;
+
+    @Inject
+    private PackageOptionRepository packageOptionRepository;
+
+    @Transactional
+    public Order order(OrderDTO orderDTO){
```

Comment:
`create()` -> returns `Order` entity which is later persisted (in controller, for example to decouple this logic from creation)